private_sharing/models.py

```diff
@@ -236,6 +236,7 @@ class DataRequestProject(models.Model):
         ),
         verbose_name="Are you requesting Open Humans usernames?",
     )
+    datatypes = models.ManyToManyField(DataTypes, related_name="data_types")
```

Comment:
I think it's unnecessary to have a related_name here, the default of `datatype_set` nomenclature seems fine to me (I like sticking to defaults as being more predictable) :)